app/src/main/res/values/strings.xml

```diff
@@ -560,7 +560,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="recommend_high_accuracy_mode">For best results, choose the High Accuracy mode.</string>
   <string name="ask_to_turn_location_on">Turn on location?</string>
   <string name="nearby_needs_location">Nearby needs location enabled to work properly</string>
-  <string name="use_location_from_similar_image">Did you shoot these two pictures at the same place? Do you want to use the latitude/longitude of the picture on the right?</string>
+  <string name="use_location_from_similar_image">Did you shoot these two pictures at the same place? Do you want to use the latitude/longitude of the picture on the right? The coordinates are not the exact coordinates, but the person who uploaded this picture thinks they close enough.</string>
```

Comment:
Hey @ujjman , thanks a lot for your first contribution. However, I am afraid that the issue is not understood completely. The issue wants to add this sentence to description of the upload. It has nothing to do with the warning to show the user. User already knows they decided to use this slightly same location. We want to make others (the ones looking at the upload) know about it.